### PR TITLE
Target

### PR DESCRIPTION
**Pull request by nasibehteimouri**
interface for target!

### CODE REVIEW

File: model/drivers/interfaces/target.py [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
-    """Description for this class"""
+class target:
+
     __metaclass__ = ABCMeta
 
     @abstractmethod
     def target(self, conf_file):
         """Creates a target"""
-        pass
+    pass
```
Comment (xuhang57) [inline model/drivers/interfaces/target.py]: we should put 4 spaces here for following the convention.

File: model/drivers/interfaces/target.py [hunk 2]
```diff
@@ -1,32 +1,31 @@
-"""This module provides interfaces for Target"""
 from abc import ABCMeta, abstractmethod
 
 
-class Target:
-    """Description for this class"""
+class target:
```
Comment (naved001) [inline model/drivers/interfaces/target.py]: Class names should normally use the CapWords convention. 

```python
class Target:
```

File: model/drivers/interfaces/target.py (hunk 1 above)
Comment (naved001) [inline model/drivers/interfaces/target.py]: 1. we don't need to put pass here at all
2. if you really want to keep it, then it should be indented to be a part of the function definition.

Comment (xuhang57) [reply model/drivers/interfaces/target.py]: I thought it would fail if we don't put pass here.

**Comment by chemistry-sourabh** [comment]
I don't want to comment in all PRs, but the interface is in the wrong place. It should be in m2/interface. Please rebase all your PRs also.

**Pull request closed without merging**